src/main/resources/db/migration/V2_67__update_vw_portefolje_info.sql

```diff
@@ -0,0 +1,71 @@
+       OB.FORMIDLINGSGRUPPEKODE          AS FORMIDLINGSGRUPPEKODE,
+       OB.ISERV_FRA_DATO                 AS ISERV_FRA_DATO,
+       OB.KVALIFISERINGSGRUPPEKODE       AS KVALIFISERINGSGRUPPEKODE,
+       OB.RETTIGHETSGRUPPEKODE           AS RETTIGHETSGRUPPEKODE,
+       OB.HOVEDMAALKODE                  AS HOVEDMAALKODE,
+       OB.SIKKERHETSTILTAK_TYPE_KODE     AS SIKKERHETSTILTAK_TYPE_KODE,
+       OB.FR_KODE                        AS FR_KODE,
+       OB.SPERRET_ANSATT                 AS SPERRET_ANSATT,
+       OB.ER_DOED                        AS ER_DOED,
+       OB.DOED_FRA_DATO                  AS DOED_FRA_DATO,
+       OD.VEILEDERIDENT                  AS VEILEDERIDENT,
+       OD.NY_FOR_VEILEDER                AS NY_FOR_VEILEDER,
+       OD.OPPFOLGING                     AS OPPFOLGING,
+       DIA.VENTER_PA_BRUKER              AS VENTERPASVARFRABRUKER,
+       DIA.VENTER_PA_NAV                 AS VENTERPASVARFRANAV,
+       BD.YTELSE                         AS YTELSE,
+       BD.UTLOPSDATO                     AS UTLOPSDATO,
+       BD.UTLOPSDATOFASETT               AS UTLOPSDATOFASETT,
+       BD.DAGPUTLOPUKE                   AS DAGPUTLOPUKE,
+       BD.DAGPUTLOPUKEFASETT             AS DAGPUTLOPUKEFASETT,
+       BD.PERMUTLOPUKE                   AS PERMUTLOPUKE,
+       BD.PERMUTLOPUKEFASETT             AS PERMUTLOPUKEFASETT,
+       BD.AAPMAXTIDUKE                   AS AAPMAXTIDUKE,
+       BD.AAPMAXTIDUKEFASETT             AS AAPMAXTIDUKEFASETT,
+       BD.AAPUNNTAKDAGERIGJEN            AS AAPUNNTAKDAGERIGJEN,
+       BD.AAPUNNTAKUKERIGJENFASETT       AS AAPUNNTAKUKERIGJENFASETT,
+       BD.NYESTEUTLOPTEAKTIVITET         AS NYESTEUTLOPTEAKTIVITET,
+       BD.AKTIVITET_START                AS AKTIVITET_START,
+       BD.NESTE_AKTIVITET_START          AS NESTE_AKTIVITET_START,
+       BD.FORRIGE_AKTIVITET_START        AS FORRIGE_AKTIVITET_START,
+       OB.TIDSSTEMPEL                    AS OPPDATERT_ARENA,
+       OD.OPPDATERT_PORTEFOLJE           AS OPPDATERT_BRUKERDATA,
+       DIA.OPPDATERT_PORTEFOLJE          AS OPPDATERT_DIALOG,
+       MAP.OPPRETTET_TIDSPUNKT           AS MAPPING_OPPRETTET,
+       OB.TIDSSTEMPEL                    AS TIDSSTEMPEL,
+       OD.MANUELL                        AS MANUELL,
+       OD.STARTDATO                      AS OPPFOLGING_STARTDATO,
+       K.RESERVASJON                     AS RESERVERTIKRR,
+       ARB.ARBEIDSLISTE_AKTIV            AS ARBEIDSLISTE_AKTIV,
+       ARB.OVERSKRIFT                    AS ARBEIDSLISTE_OVERSKRIFT,
+       ARB.KOMMENTAR                     AS ARBEIDSLISTE_KOMMENTAR,
+       ARB.FRIST                         AS ARBEIDSLISTE_FRIST,
+       ARB.SIST_ENDRET_AV_VEILEDERIDENT  AS ARBEIDSLISTE_ENDRET_AV,
+       ARB.ENDRINGSTIDSPUNKT             AS ARBEIDSLISTE_ENDRET_TID,
+       ARB.KATEGORI                      AS ARBEIDSLISTE_KATEGORI,
+       VS.VEDTAK_STATUS_ENDRET_TIDSPUNKT AS VEDTAK_STATUS_ENDRET_TIDSPUNKT,
+       VS.VEDTAKSTATUS                   AS VEDTAKSTATUS,
```

Comment:
i think that ANSVARLIG_VEILEDER should be added here `VS. ANSVARLIG_VEILEDER  AS ANSVARLIG_VEILEDER`

Reply:
Thanks a lot! I missed that